packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -395,6 +406,8 @@ export function sendTransaction<
 					) {
 						transactionFormatted = {
 							...transactionFormatted,
+							// TODO gasPrice, maxPriorityFeePerGas, maxFeePerGas
```

Comment:
This is a point of discussion, `getTransactionGasPricing` is going to return an object:

```typescript
{
    gasPrice: HexString,
    maxPriorityFeePerGas: HexString,
    maxFeePerGas: HexString,
}
```

and either `gasPrice` or `maxPriorityFeePerGas` and `maxFeePerGas` are going to be `undefined`. So unpacking the object (`...{}`) results in the `undefined` property being added to the transaction object that gets submitted to the RPC node. This is generally not going to be a problem for most users, but it personally doesn't sit right with me to add unnecessary undefined properties to a user's object before submission - the library should modify user data the least amount

Reply:
This also means that the transaction object that gets emitted for `sending` and `sent` PromiEvents will not match what the user provided as input to the method